api/api.go

```diff
@@ -62,7 +62,11 @@ const (
 	apiAccountOrganizationInfo = "external/account/organizationInfo"
 
 	// Alpha
-	apiLQLQuery = "external/lql/query"
+	apiLQL            = "external/lql"
+	apiLQLCompile     = "external/lql/compile"
+	ApiLQLDataSources = "external/lql/dataSources"
```

Comment:
Why this changed to be exported? 
```suggestion
	apiLQLDataSources = "external/lql/dataSources"
```

Reply:
So that I can access this via `api.ApiLQLDataSources` in my test file:

```
func TestDataSourcesMethod(t *testing.T) {
	fakeServer := lacework.MockServer()
	fakeServer.MockAPI(
		api.ApiLQLDataSources,
```